Library/EnvelopeFactory.ts

```diff
@@ -1,13 +1,163 @@
+import os = require("os");
 import Contracts = require("../Declarations/Contracts")
 import Util = require("./Util")
 import Config = require("./Config");
 import Context = require("./Context");
 import { CorrelationContextManager } from "../AutoCollection/CorrelationContextManager";
 
+var StreamId = Util.w3cTraceId(); // Create a guid
+
 /**
  * Manages the logic of creating envelopes from Telemetry objects
  */
 class EnvelopeFactory {
+    public static createQuickPulseEnvelope(metrics: Contracts.MetricQuickPulse[], documents: Contracts.DocumentQuickPulse[], config: Config, context: Context): Contracts.EnvelopeQuickPulse {
```

Comment:
Should these have been put into a separate class? Eg. QuickPulseEnvelopeFactory?

Reply:
Moved to QuickPulseEnvelopeFactory